pkgdb2/api/extras.py

```diff
@@ -216,7 +218,10 @@ def api_bugzilla():
     acls = _bz_acls_cached(name, out_format)
 
     if out_format == 'json':
-        return flask.jsonify(acls)
+        return flask.Response(
+            str(acls).replace("'", '"').replace('None', 'null'),
```

Comment:
On Wed, Jan 07, 2015 at 08:25:56AM -0800, Ralph Bean wrote:
>    In pkgdb2/api/extras.py:
> 
>  > @@ -216,7 +218,10 @@ def api_bugzilla():
>  >      acls = _bz_acls_cached(name, out_format)
>  > 
>  >      if out_format == 'json':
>  > -        return flask.jsonify(acls)
>  > +        return flask.Response(
>  > +            str(acls).replace("'", '"').replace('None', 'null'),
> 
>    Also, this seems error prone in general.
> 
>      * Did you try import json; json.dumps(acls)? Was it faster than
>        flask.jsonify(acls)?

That's what flask.jsonify is doing:
https://github.com/mitsuhiko/flask/blob/master/flask/json.py#L201

>      * What if there is a single ' in some string? Replacing it with " would
>        be incorrect.

Since the output only contains package name, FAS username and FAS groups, I
think we're safe on this.

>      * When converting the string to json, true you need to convert None to
>        null, but you also need to convert True to true and False to false.

jsonlint.com seems to agree with you, I wonder why the unit-tests passed then
:-/